nltk/parse/corenlp.py

```diff
@@ -105,7 +105,12 @@ def __init__(
         self.corenlp_options = corenlp_options
         self.java_options = java_options or ['-mx2g']
 
-    def start(self):
+    def start(self, stdout='devnull', stderr='devnull'):
+        """ Starts the CoreNLP server
+
+        :param stdout: Specifies the subprocess's loc Either 'devnull', 'stdout' or 'pipe'
```

Comment:
These might need to be rewritten, to something like:
```
Specifies where CoreNLP output is redirected. Valid values are 'devnull', 'stdout', 'pipe'.
```

Maybe, it worth explaining what the intention of the  `devnull` value is, as it might not be clear.

Reply:
I also agree with this! 😄